tests/test_isrTask.py

```diff
@@ -340,6 +339,25 @@ def test_overscanCorrection(self):
         statAfter = computeImageMedianAndStd(self.inputExp.image[self.amp.getRawDataBBox()])
         self.assertLess(statAfter[0], statBefore[0])
 
+    def test_overscanCorrectionMedianPerRow(self):
```

Comment:
Like we discussed on Slack, this is a great start. Could you extend it to putting in the sine wave and checked that after doing the correction things are reasonably flat though? I think that would be a great check.

Reply:
(and especially important if this is all going to be refactored soon)

Reply:
We discussed that we could add a fake sinusoidal signal when we thought that this function was the only test for overscan correction. But then I saw that there are more tests in `test_overscanCorrection.py` that create mock images with mock overscans (10x10 imaging area full of 10's, with a 10x3 overscan region full of 2's), and I thought that creating the sine wave was not necessary anymore (these tests pass).  Would you like to see another test that, instead of the 2's in the overscan, has values that follow a sine wave?

Reply:
Yeah, I think that would be good. Just checking the image median and std-dev are lower is a _really_ low bar; doesn't just doing `image = image/2` fulfill that?! (Median and stddev both go down there)